### ISSUE TITLE
kubejs registries rewrite broke this for 1.19.2

### ISSUE THREAD

**Issue by Arutheed** [screret/beJS#4]
The kubejs registries rewrite in may broke BEJSPlugin for 1.19.2
Screenjs seems to be effected aswell.

**Comment by s-lach** [comment]
I encountered the same problem. Any ideas?

**Comment by LincStoel** [comment]
After beating my head against this for hours i found this and it appears that this mod is still broken. I am consistently getting "unknown type" errors and " no builders" errors. I thought it was my code for sure, but seeing this and the fact that the examples don't work when i paste them has really reinforced that i don't think I'm the problem here.

**Comment by ExNihilius** [comment]
Yeah, since this mod was last worked on kubeJS had a LOT of updates/changes.
Apart from forking it and putting a lot of work into reworking it, I don't see this being usable anytime soon.
I believe Screret is quite busy with other projects atm.
